tests/wpt/web-platform-tests/html/semantics/embedded-content/the-iframe-element/stash.py

```diff
@@ -1,6 +1,6 @@
 # This Source Code Form is subject to the terms of the Mozilla Public
 # License, v. 2.0. If a copy of the MPL was not distributed with this
-# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
```

Comment:
I would rather not make any changes to upstream WPT files as part of our HTTP->HTTPS work. Let's revert any changes under tests/wpt/web-platform-tests.